packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -0,0 +1,229 @@
+// @flow
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Dropdown from "./dropdown.js";
+import type {Item, DropdownItem} from "../util/types.js";
+import ActionItem from "./action-item.js";
+import OptionItem from "./option-item.js";
+import DropdownAnchor from "./dropdown-anchor.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /*
+     * The callback function that's passed the openDropdown function, used to rerender
+     the opener
+     */
+    children: (openDropdown: () => void) => React.Element<any>,
```

Comment:
This should look something like:
```
children: (openDropdown: () => void, state: ClickableState) => React.Element<any>,
```
where `ClickableState` is defined as:
```
type ClickableState = {
    pressed: boolean,
    hovered: boolean,
    focused: boolean,
};
```
We don't have a `ClickableState` type defined anywhere but we probably should.

We may also want to pass the handlers too.  If we don't then we'll need to prevent `children` from returning a `React.Fragment`.  It's possible that `React.Element<any>` already does this.